imports/plugins/core/checkout/client/containers/filledCartDrawerContainerCustomer.js

```diff
@@ -0,0 +1,98 @@
+import { compose, withProps } from "recompose";
+import { $ } from "meteor/jquery";
+import { Session } from "meteor/session";
+import { registerComponent, composeWithTracker } from "@reactioncommerce/reaction-components";
+import { Reaction } from "/client/api";
+import withUpdateCartItemsQuantity from "/imports/plugins/core/graphql/lib/hocs/withUpdateCartItemsQuantity";
+import FilledCartDrawer from "../components/filledCartDrawer";
+
+// event handlers to pass in as props
+const handlers = {
+  handleImage(item) {
+    return (item && item.imageURLs) ? item.imageURLs.small : null;
+  },
+
+  handleShowProduct(productItem) {
+    if (productItem) {
+      Reaction.Router.go("product", {
+        handle: productItem.productSlug,
+        variantId: productItem.productConfiguration.productVariantId
+      });
+    }
+  },
+
+  pdpPath(productItem) {
+    if (productItem) {
+      const handle = productItem.productSlug || (productItem.productConfiguration && productItem.productConfiguration.productId);
+      return Reaction.Router.pathFor("product", {
+        hash: {
+          handle,
+          variantId: productItem.variantId
+        }
+      });
+    }
+    return "";
+  },
+
+  handleCheckout() {
+    document.querySelector("#cart-drawer-container").classList.remove("opened");
+    Session.set("displayCart", false);
+    return Reaction.Router.go("cart/checkout");
+  }
+};
+
+/**
+ * @name composer
+ * @private
+ * @summary Subscribes to images for products in cart & passes cart products to CartDrawer
+ * @param {Object} props - React props
+ * @param {Function} onData - Function to call when data is ready
+ * @returns {undefined}
+ */
+function composer(props, onData) {
+  const { cartItems, cartId } = props;
+  if (!cartItems) return;
+
+  const handleRemoveItem = (event, item) => {
+    event.stopPropagation();
+    event.preventDefault();
+    const cartItemElement = $(event.target).closest(".cart-drawer-swiper-slide");
```

Comment:
Maybe use pure js instead of depending on jquery so it can be ported into components library later if needed?